app/javascript/components/EditForm.js

```diff
@@ -15,47 +14,69 @@ import { IncomeDeclaration } from "./IncomeDeclaration";
 const databaseURL = process.env.APP_URL  
```

Comment:
why did you name this 'databaseUrl'?

Reply:
it is how I would normally do it for a separate react app but its true that in this case it is part of the same app, have changed it